README.md

```diff
@@ -9,22 +9,44 @@ npm install -g tattoo
 
 ### Usage
 
-***Test all repos***
-```
-mkdir tattoo_work
-cd tattoo_work
-tattoo
-```
+When you run tattoo, it will create and/or reuse a directory called
+`./tattoo_workspace`, relative to the current working dir.  This folder is
+used to clone test target repositories and their dependencies.  You can change
+the directory used with the `--workspace-dir` or `-w` option.
 
 ***Test one repo***
 ```
-tattoo -t paper-input --verbose
+tattoo PolymerElements/paper-input
```

Comment:
Good question.  This README didn't to explain what the tool is actually doing.  I've updated it 380f4b955d2eb3f77095d4060ec421ecf3ccb546